app/model/TeacherModel.php

```diff
@@ -38,17 +38,29 @@ public function deleteById(string $id): void {
         $this->database->query('DELETE FROM SEM_UCITEL WHERE ID=?', $id);
     }
 
-    public function insert(array $item): void {
+    public function insert(array $item): string {
+        $id = null;
+        $statement = $this->database->getPdo()->prepare('INSERT INTO SEM_UCITEL (id, jmeno, prijmeni, titul_pred, titul_za, telefon, mobil, kontaktni_email, katedra_id) VALUES (SEM_UCITEL_SEQ.NEXTVAL, ?, ?, ?, ?, ?, ?, ?, ?) RETURNING id INTO ?');
+        $statement->bindParam(1, $item['firstName']);
+        $statement->bindParam(2, $item['lastName']);
+        $statement->bindParam(3, $item['prefixTitle']);
+        $statement->bindParam(4, $item['postfixTitle']);
+        $statement->bindParam(5, $item['telephone']);
+        $statement->bindParam(6, $item['mobile']);
+        $statement->bindParam(7, $item['email']);
+        $statement->bindParam(8, $item['department']);
+        $statement->bindParam(9, $id, \PDO::PARAM_INT, 8);
+
+        $statement->execute();
```

Comment:
Nevím jak to je tady přesně stavěný, ale kdyžtak jde do execute() dát přímo pole parametrů (ať už podle indexu (`[0 => value]`), nebo podle jména (`[':parameter' => value]`) bez pro každý vypisovat bindParam.

Reply:
👍 Uz to delat nejdu, ale jn... Jsem puvodne myslel ze je pojmenuju

Reply:
Ale hlavne jsem potreboval ten posledni predat jako spešl referenci... aby mi to tam dalo hodnotu a ne opacne